js/viewer/annotation/base.js

```diff
@@ -6,16 +6,25 @@ function Base(attrs){
     this.isDrawing = false;
     this.constants = attrs.parent.parent.parent.all_config.constants;
     this.urlParams = attrs.parent.parent.parent.parameters;
+    this.annotationUtils = new AnnotationUtils();
```

Comment:
Since you moved all the logic to be in the annotation-svg, you don't need the utils here anymore.

Reply:
Removed